static/css/jira-configuration.css

```diff
@@ -408,6 +408,23 @@
   max-width: 332px;
 }
 
+.jiraConfiguration__table__editContainer,
+.jiraConfiguration__syncRetryModal__closeBtn {
+  border-radius: 3px;
+  cursor: pointer;
+  padding: 0.45em;
+  position: relative;
+}
+.jiraConfiguration__table__editContainer:hover,
+.jiraConfiguration__syncRetryModal__closeBtn:hover {
+  background: #091E4208;
+}
+
+.learn-more-link {
+  display: inline-block;
+  margin-right: 12px;
```

Comment:
I think we need to decrease the spacing here. See the screenshot attached. It's a screenshot of what you uploaded for the gh config page overlaid with the buttons from Jade's design spec.

<img width="597" alt="Screen Shot 2022-06-02 at 11 20 47 am" src="https://user-images.githubusercontent.com/37155488/171528034-74705f9c-41cf-4d33-8bb9-d14625d2d4cb.png">

Reply:
Margin value reduced to `4px`